src/mouse/segmentation/optimisation.py

```diff
@@ -24,7 +24,8 @@
     "_balloon_latent": (0.2, 1.8),
     "sigma": (1, 10),
 }
-GAC_SEARCH_SPACE_TUNE = {k: tune.sample.Float(*v) for k, v in GAC_SEARCH_SPACE.items()}
+# TODO check if the tune still works
```

Comment:
what does it mean ?

Reply:
done

Reply:
what do you mean by done ?

Reply:
we create an issue with this previous todo, to look at it in the future